test/bench/alloc.mo

```diff
@@ -16,7 +16,7 @@ actor alloc {
         };
         let (m1, n1) = counters();
         debugPrint(debug_show (m1 - m0, n1 - n0));
-
+        assert false; // rollback changes
```

Comment:
Is `drun` suffering from a timeout? Maybe we could also just downsize the test. I believe not rolling back could maybe also be interesting to assess the GC time (with a growing heap).

Reply:
Yes, I think I added the rollback when I only had 24 bits compact bignums, leading to loads of boxing. Probably safe to revert now we have 30 bit compact bignums.

Reply:
Reverted

Reply:
Reverted